frontend/app/model/ResponsiveImage.scala

```diff
@@ -7,7 +7,7 @@ case class ResponsiveImageGroup(
   altText: String,
   availableImages: Seq[ResponsiveImage]
 ) {
-  val defaultImage = availableImages.head.path
+  val defaultImage = availableImages.headOption.map(_.path)
```

Comment:
@feedmypixel This might conflict as I've just changed this to `val defaultImage = sortedImages.find(_.width > 300).map(_.path).getOrElse(smallestImage)` in PR https://github.com/guardian/membership-frontend/pull/312